tardis/model/parse_input.py

```diff
@@ -678,6 +679,13 @@ def parse_csvy_radiation_field_state(
             geometry, packet_source
         )
 
+    if np.any(t_radiative < 1000 * u.K):
+        warnings.warn(
```

Comment:
please use logging.warn (and remove the warnings import)

Reply:
Done.